### PR TITLE
Airtel tigo sms service

### PR DESCRIPTION
**Pull request by kenessajr**
Added 63513 and 63514 SMS services, also worked on a standardize lower case attribute within most used attribute internally such `country name`, `service name, action` and `telco name`. 
Added API Creds to `.flaskenv` file.

### CODE REVIEW

File: api/.flaskenv [hunk 1]
```diff
@@ -14,4 +14,6 @@ MAIL_USERNAME=eugene@pindo.io
 MAIL_PASSWORD=r7BRjuvy5&ZF
 SECURITY_PASSWORD_SALT=my_password_salt
 
-TELCO_63510_SMS_SEND_PASSWORD=3PiNBq
\ No newline at end of file
+TELCO_63510_SMS_SEND_PASSWORD=3PiNBq
```
Comment (erwagasore) [inline api/.flaskenv]: Please find a way to not share passwords of services and for email too.

File: api/app/sms/v1/test_sms.py [hunk 2]
```diff
@@ -131,3 +131,33 @@ def test_messages_routes(self, db_session, user, superuser, credentials, super_c
             url_for('sms_v1.list'), headers=super_credentials, content_type='application/json')
         assert resp.status_code == 200
         assert len(resp.get_json()['sms']) == 0
+
+        # Create Tigo and Airtel service
+        tigo_rw = Telco(name='TIGO', mnc='13', country=rwanda)
```
Comment (erwagasore) [inline api/app/sms/v1/test_sms.py]: For example on the creation of "TIGO" do we store "tigo" as name?

File: api/app/countries/test_countries.py [hunk 3]
```diff
@@ -34,7 +34,7 @@ def test_countries_routes(self, db_session, credentials, super_credentials):
         assert Country.query.count() == 1
 
         # check if the country is commited to the database
-        country = Country.query.filter_by(name='Rwanda').first()
+        country = Country.query.filter_by(name='rwanda').first()
```
Comment (erwagasore) [inline api/app/countries/test_countries.py]: Please test also cases where you created a country with all the characters capitalized and check if the stored value for name is different.

File: api/app/sms/v1/test_sms.py (hunk 2 above)
Comment (kenessajr) [reply api/app/sms/v1/test_sms.py]: Yes, we store in the database tigo as the name.

File: api/app/countries/test_countries.py (hunk 3 above)
Comment (kenessajr) [reply api/app/countries/test_countries.py]: ok

File: api/.flaskenv (hunk 1 above)
Comment (kenessajr) [reply api/.flaskenv]: @erwagasore I'm not finding a proper way to do it, I think we can start exploring Kubernates now for fixing the issue with passwords.

**Pull request closed without merging**